### PR TITLE
Removes Dauntless Self Destruct

### PR DESCRIPTION
**Pull request by StrangeWeirdKitten**
## About The Pull Request

Removes the Dauntless self destruct and the dense rock surrounding the base. Sets the exterior turrets to neutral.

## Why It's Good For The Game

No one uses it anyway and I've come to the conclusion that it's not really needed. The dense rock primarily existed to stop a stray colossus bolt from detonating the entire base. With the removal of the dense rock, the exterior turrets will only target fauna except these two hanger turrets (these will still shoot you)

image

Also it's not like it's not super easy to cheese the already existing turrets with the dense rock or a few selective mined out rocks. They're toned back signifigantly because Dauntless is chunky, and I'm not going to make ~ 60 x 70 area of lavaland (base + turret range) automatic kill on sight if you accidently stray a bit too close to the three shot turrets. 

## Proof Of Testing

It compiles and works on the live server.

## Changelog

:cl:
del: Removes Dauntless self destruct and dense rock surrounding the base
balance: Dauntless exterior turrets shoot fauna, not people
/:cl:

### CODE REVIEW

File: modular_zubbers/maps/offstation/dauntless/mob_spawns.dm [hunk 1]
```diff
@@ -40,7 +40,7 @@
 	prompt_name = "a Syndicate hostage"
 	you_are_text = "You are a hostage onboard an unknown syndicate vessel"
 	flavour_text = "Unaware of where you are, all you know is you are a prisoner. The plastitanium should clue you into who your captors are... as for why you're here? That's for you to know, and for us to find out."
-	important_text = "You are not an antagonist. You are still bound to the Roleplay Rule 4. Dauntless personnel can throw you into lava if you antagonize them"
+	important_text = "You are not an antagonist. You are still bound to the Roleplay Rules regarding intent and esculation. Dauntless personnel can throw you into lava if you antagonize them."
```
Comment (ReturnToZender) [inline modular_zubbers/maps/offstation/dauntless/mob_spawns.dm:43]:
```suggestion
	important_text = "You are not an antagonist. You are still bound to the Roleplay Rules regarding escalation. Dauntless personnel can throw you into lava if you antagonize them."
```

Comment (ReturnToZender) [reply modular_zubbers/maps/offstation/dauntless/mob_spawns.dm:43]: Spellcheck!

**Review by Majkl-J — COMMENTED** [review]
Bit iffy on the turrets and walls since they also make it significantly harder for miners to create conflict with the base at their own volition, something that they still sometimes manage even with how the current base is set up

**Comment by StrangeWeirdKitten** [comment]
> Bit iffy on the turrets and walls since they also make it significantly harder for miners to create conflict with the base at their own volition, something that they still sometimes manage even with how the current base is set up

Interdyne has always had neutral turrets for months, if not years. There is still two shot anyone turrets at the actual hanger 

> Bit iffy on the turrets and walls since they also make it significantly harder for miners to create conflict with the base at their own volition, something that they still sometimes manage even with how the current base is set up

Interdyne (before dome) worked like this for months with very little issues.

**Comment by AllCorruptingWyrm** [comment]
Megafauna will just aggro on the hanger or sometimes (rarely) spawn around the dauntless

**Pull request merged**